src/QEQ/fix_qeq.h

```diff
@@ -100,6 +100,11 @@ class FixQEq : public Fix {
 
   double alpha;
 
+  // ctip
+
+  double cdamp;
```

Comment:
Move to child class.

Reply:
done. also for "alpha" which belongs to qeq/slater